packages/fee-estimation/src/estimateFees.ts

```diff
@@ -58,7 +58,7 @@ const knownChains = [
 type ClientOptions =
   // for known chains like base don't require an rpcUrl
   | {
-      chainId: typeof knownChains[number]
+      chainId: (typeof knownChains)[number]
```

Comment:
weird, why are we seeing this locally, if it wasn't an issue for the original repository?

Reply:
I think `pnpm lint:fix` did this.